src/main/java/me/zeroeightsix/kami/command/commands/SayCommand.java

```diff
@@ -0,0 +1,26 @@
+package me.zeroeightsix.kami.command.commands;
+
+import me.zeroeightsix.kami.command.Command;
+import me.zeroeightsix.kami.command.syntax.ChunkBuilder;
+
+/**
+ * @author S-B99
+ * Created by S-B99 on 12/03/20
+ */
+public class SayCommand extends Command {
```

Comment:
The whole purpose of this command suggests there is a problem in your command parsing. Why not for example allow a prefix to be prepended with `\` in order to prevent it being parsed as a command?

Reply:
Actually this already works, you're right, it was completely unnecessary.